app/controllers/provider/admin/cms/files_controller.rb

```diff
@@ -48,11 +48,7 @@ def destroy
   private
 
   def file_params
-    params[:cms_file].dup.tap do |params|
-      if section = params[:section_id]
-        params[:section] = current_account.sections.find(section)
-      end
-    end
+    params.require(:cms_file).permit(:path, :attachment, :downloadable, :tag_list, :section_id)
```

Comment:
Same as https://github.com/3scale/porta/pull/3228/files#r1124388383

Can set the section that is from a different tenant.